roles/openshift_monitor_availability/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+openshift_monitor_app_create_image: quay.io/redhat/openshift-monitor-project-lifecycle:9909bbb
```

Comment:
openshift-enterprise should specify a precise version that tracks with the release of openshift.

Reply:
work being delivered out of quay right now does not track openshift-enterprise versioning.

see: https://github.com/openshift/openshift-ansible/pull/7693/files#diff-d9705c01debfd746d4d97a83d7b42759R16

we should get the work to track a tag, but that should not hold up this pr.

Reply:
btw, we're now [tagging the cited images with semvers](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_cluster_monitoring_operator/defaults/main.yml#L2), at least. I'd like to do the same thing here, soon, in a followup

Reply:
I'm trying to keep all of the extra components to some baseline level of uniformity.  The "Don't worry we'll get to it" is too easy to fall through the cracks and then we're shipping code we forgot to update.

I don't think it's a big ask to tag in accordance with the other components.